src/css/_coa_variables.scss

```diff
@@ -6,8 +6,10 @@ $coa-space-level1: 5px;
 $coa-space-level2: 10px;
 $coa-space-level3: 15px;
 $coa-space-level4: 20px;
-$coa-space-level5: 40px;
-$coa-space-level6: 60px;
+$coa-space-level5: 25px;
+$coa-space-level6: 30px;
+$coa-space-level7: 40px;
+$coa-space-level8: 60px;
```

Comment:
How are we deciding these spacings? Should they be px? or rem?
Systems I've liked in the past do something like:
```css
coa-spacing: 10px;
coa-spacing-half: 5px;
coa-spacing-times2: 20px;
coa-spacing-times3-half: 35px;
```
that way the number units stay in sync. 🤷‍♂️

Reply:
I NEEEEEEED to convert these to rems for sure. I picked this naming convention to keep the name detached from the value, so if we change our minds in the future, I just have to update a rem value, and not all the variable names and their references. Also, I'm pulling these values from spacings in Celine's mocks. 

Honestly, I think space-xs, space-s, space-m, space-l, space-xl is the real way to go. This will also force us to limit the variety of spacings to only 5 which will keep things visually consistent. I just haven't thought much of it since it seems like the mocks are still in flux.

Reply:
yeah, tshirt size FTW! lets try that!